src/client/ua_client.c

```diff
@@ -31,6 +32,36 @@ clientHouseKeeping(UA_Client *client, void *_);
 /* Client Lifecycle */
 /********************/
 
+UA_StatusCode
+UA_ClientConfig_copy(UA_ClientConfig const *src, UA_ClientConfig *dst){
+    UA_StatusCode retval = UA_STATUSCODE_GOOD;
```

Comment:
Run clientconfig_clear when copying fails.
Attention! This might clear strings that are also in the original config.

Also make ClientConfig_clear public.
So we can work with stack-allocated configs.

Reply:
👍